Log message:
atyfb_driver is protected with CONFIG_PCI

Modified files:
    linux/drivers/video/aty:
	atyfb_base.c 


Index: linux/drivers/video/aty/atyfb_base.c
Stats: 4 insertions
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/drivers/video/aty/atyfb_base%2ec.diff?r1=1.1.1.29&r2=1.6
========================================================================
--- linux/drivers/video/aty/atyfb_base.c	9 Apr 2006 16:56:04 -0000	1.1.1.29
+++ linux/drivers/video/aty/atyfb_base.c	9 May 2006 18:49:14 -0000	1.6
@@ -3719,7 +3719,9 @@
     atyfb_setup(option);
 #endif
 
+#ifdef CONFIG_PCI
     pci_register_driver(&atyfb_driver);
+#endif
 #ifdef CONFIG_ATARI
     atyfb_atari_probe();
 #endif
@@ -3728,7 +3730,9 @@
 
 static void __exit atyfb_exit(void)
 {
+#ifdef CONFIG_PCI
 	pci_unregister_driver(&atyfb_driver);
+#endif
 }
 
 module_init(atyfb_init);
-
To unsubscribe from this list: send the line "unsubscribe linux-m68k-cvscommit" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

